owl/SOMA-ACT.owl

```diff
@@ -361,13 +392,24 @@ However, sometimes this might be not the case, e.g., with counterquestions."@en)
 AnnotationAssertion(rdfs:label :QueryAnsweringTask "Query answering task")
 EquivalentClasses(:QueryAnsweringTask ObjectIntersectionOf(:AnsweringTask ObjectAllValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies> ObjectAllValuesFrom(:isReactionTo ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isClassifiedBy> SOMA:QueryingTask)))))
 
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#AbductiveReasoning> (Abductive reasoning)
+# Class: :Serving (:Serving)
+
+AnnotationAssertion(rdfs:comment :Serving "The task in which the agent delivers food"^^xsd:string)
+SubClassOf(:Serving SOMA:Delivering)
+SubClassOf(:Serving ObjectAllValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies> ObjectIntersectionOf(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Action> ObjectUnionOf(ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasParticipant> SOMA:Dish) ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasParticipant> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAgent>)))))
```

Comment:
I don't think serving always involves a dish, e.g. I could serve a drink.
Maybe better specify the roles of the task? Then you can also have "unusual" objects playing a role in the task. E.g. maybe a stone-biter serves a stone to another stone biter ;)